sematic/api/endpoints/runs.py

```diff
@@ -427,6 +429,21 @@ def save_graph_endpoint(user: Optional[User]):
     return flask.jsonify({})
 
 
+@sematic_api.route("/api/v1/runs/<run_id>/external_resources", methods=["GET"])
+@authenticate
+def get_run_external_resources(user: Optional[User], run_id):
+    with db().get_session() as session:
+        result = session.query(RunExternalResource, ExternalResource) \
+            .filter(RunExternalResource.run_id == run_id) \
+            .join(ExternalResource,
+                  ExternalResource.id == RunExternalResource.resource_id) \
+            .all()
```

Comment:
We typically store these in `sematic.db.queries`. Can you create the following function:
```python
def get_run_external_resources(run_id: str) -> List[ExternalResource]:
```

Reply:
Sure. Implemented in `sematic.db.queries`. Name was slightly changed to avoid a collision.